src/ceph-disk

```diff
@@ -711,7 +711,7 @@ def get_conf(cluster, variable):
                 'ceph-conf',
                 '--cluster={cluster}'.format(
                     cluster=cluster,
-                    ),
+                    ) if cluster != None else '',
```

Comment:
can this really happen ? It looks like this will replace --cluster with an argument that is the empty string and that's going to fail. Or am I misreading this ?